RecoTauTag/HLTProducers/src/L1HLTTauMatching.cc

```diff
@@ -31,8 +31,8 @@ void L1HLTTauMatching::produce(edm::StreamID iSId, edm::Event& iEvent, const edm
 
   unique_ptr<PFTauCollection> tauL2jets(new PFTauCollection);
 
-  double deltaR = 1.0;
-  double matchingR = 0.5;
+  constexpr double matchingR = 0.5;
```

Comment:
```suggestion
  constexpr double matchingR2 = 0.5;
```

Reply:
Here there was another mistake of mine:
```suggestion
  constexpr double matchingR2 = 0.5 * 0.5;
```